### PR TITLE
[artifact-manager,installer,maven] (#328) Remove `vrang.project.id` configuration property

### PR DESCRIPTION
**Pull request by Michaelpalacce**
### Description

`vrang.project.id` has been entirely deprecated and instead `vrang.project.name` should be used

### Checklist

- [x] I have added relevant error handling and logging messages to help troubleshooting
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation, relevant usage information (if applicable)
- [x] I have updated the PR title with affected component, related issue number and a short summary of the changes introduced
- [x] I have added labels for implementation kind (kind/*) and version type (version/*)
- [x] I have tested against live environment, if applicable
- [x] I have synced any structure and/or content vRA-NG improvements with vra-ng and ts-vra-ng archetypes (if applicable)
- [x] I have my changes rebased and squashed to the minimal number of relevant commits. **Notice: don't squash all commits**
- [x] I have added a descriptive commit message with a short title, including a `Fixed #XXX -` or `Closed #XXX -` prefix to auto-close the issue

### Testing

**Pending**

### Release Notes

**Pending**

### Related issues and PRs

- #328

### CODE REVIEW

**Comment by VenelinBakalov** [comment]
Looks good, I guess we can remove `org.id` as well in a separate PR to make things more strucutred?

**Comment by Michaelpalacce** [comment]
> Looks good, I guess we can remove `org.id` as well in a separate PR to make things more strucutred?

Ahm, actuallyyyyy I forgot about that one 😄 Yeah, lets do it in a separate PR

File: common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/configuration/ConfigurationVraNg.java [hunk 1]
```diff
@@ -263,7 +263,8 @@ public void validate(boolean domainOptional) throws ConfigurationException {
 	public void deprecationWarnings() {
 		String[] deprecatedFlags = new String[] {
 				"bp.ignore.versions",
-				"bp.release"
+				"bp.release",
+				"project.id"
```
Comment (VenelinBakalov) [inline common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/configuration/ConfigurationVraNg.java:267]: Good catch!

**Review by VenelinBakalov — CHANGES_REQUESTED** [review]
I think we need to also remove `vrang_org_id` for the `installer`

File: docs/versions/latest/Release.md [hunk 2]
```diff
@@ -120,3 +126,9 @@ You don't need to migrate projects if they are `abx` based. `nodejs` is the corr
 ### *Migrate PolicyTemplates*
 
 Search your projects that use `@PolicyTemplate` decorator. The `templateVersion` property is now required. Check on top for possible values
+
+### *Migrate away from `vrang.project.id`*
```
Comment (VenelinBakalov) [inline docs/versions/latest/Release.md:130]: We might need a similar section for `vrang_org_id` and `installer` file, e.g. `env.properties`

File: common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/configuration/ConfigurationVraNg.java (hunk 1 above)
Comment (Michaelpalacce) [reply common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/configuration/ConfigurationVraNg.java:267]: ikr! 😄

File: docs/versions/latest/Release.md (hunk 2 above)
Comment (Michaelpalacce) [reply docs/versions/latest/Release.md:130]: Will open a new PR

**Comment by Michaelpalacce** [comment]
@VenelinBakalov Opened a new pr for org id, can you review again?

**Review by VenelinBakalov — APPROVED** [review]

**Review by VenelinBakalov — APPROVED** [review]

**Pull request merged**